api/IDBCursorWithValue.json

```diff
@@ -52,10 +52,10 @@
             "version_added": "14"
           },
           "safari": {
-            "version_added": "7"
+            "version_added": "6.1"
```

Comment:
I can confirm that this and other bits of IndexedDB were exposed in Safari 6.1 using BrowserStack, but I think we should change everything listed by `npm run traverse safari api 7 | grep -iE 'idb|indexeddb'` in one go if we're going to do it, and that would require care to not change things that actually didn't ship in the initial release. https://caniuse.com/indexeddb says something pretty different here, with notes about very buggy behavior.

Can you separate all changes that decrease the version number, bump any added "6.1" to "7" and  file an issue for this particular mess?

Reply:
By running the traverse script, it appears the only IDB entries still marked as "7" is the `IDBCursor` (which I might have accidentally undone the changes for; running the collector in Safari 6.1 confirms support) and various events (which the collector doesn't update anyways).  But for Safari iOS, support was not added until iOS 8, as I have just confirmed, so I'll be reverting the Safari updates.